### PR TITLE
make spectrogram augmenter optional

### PR DESCRIPTION
**Pull request by zhiyun**

### CODE REVIEW

File: axlearn/audio/asr_encoder_test.py [hunk 1]
```diff
@@ -28,6 +29,7 @@ class SpeechFeatureLayerTest(TestCase):
     """Tests SpeechFeatureLayer."""
 
     @parameterized.parameters([True, False])
+    @pytest.mark.fp64
```
Comment (markblee) [inline axlearn/audio/asr_encoder_test.py:32]: I wonder why you see these changes? Is this rebased to main?

Comment (zhiyun) [reply axlearn/audio/asr_encoder_test.py]: Now solved.

**Comment by madrob** [comment]
You may need to rebase your changes to get CI to run. Thanks!

**Comment by zhiyun** [comment]
> You may need to rebase your changes to get CI to run. Thanks!

Thanks Mike. Hmm I'm confused on why ci is not kicked off...

**Review by markblee — APPROVED** [review]

**Pull request closed without merging**